### ISSUE TITLE
Unable to see BLE when running BLE_LED demo

### ISSUE THREAD

**Issue by alistair23** [sparkfun/mbed-os-ambiq-apollo3#4]
### Description of defect

I understand that BLE is currently under development. I couldn't figure out what the current state is so I thought I would open this issue.

I am able to build and flash the [`BLE_LED` example for mbed](https://github.com/ARMmbed/mbed-os-example-ble/tree/master/BLE_LED) on the [SparkFun RedBoard Artemis Nano](https://www.sparkfun.com/products/15443) using the ambiq-apollo3-2 branch. The example expects two LEDs, while the board only has 1 exposed so a small change is required to allow it to compile.

The example starts and I see a message on the UART with the Bluetooth MAC address. I am then unable to see the Bluetooth connection from any other Bluetooth device. I have tried following the [steps from the example](https://github.com/ARMmbed/mbed-os-example-ble/tree/master/BLE_LED#requirements) but am still unable to see the device.

#### Target(s) affected by this defect ?

[SparkFun RedBoard Artemis Nano](https://www.sparkfun.com/products/15443)

#### Toolchain(s) (name and version) displaying this defect ?

`arm-none-eabi-gcc (Arch Repository) 9.3.0`

#### What version of Mbed-os are you using (tag or sha) ?

The ambiq-apollo3-2 branch, SHA: f9bd2afc123b0e8f7f2f30efd8657094030d26da

#### What version(s) of tools are you using. List all that apply (E.g. mbed-cli)

mbed-cli: version 1.10.2

#### How is this defect reproduced ? 

Try to run the BLE examples and see if you can see the SparkFun RedBoard Artemis Nano from another BLE device.

**Comment by dwebb72** [comment]
Did you ever get this to work? I am in the same position now.

**Comment by alistair23** [comment]
Nope. I made some BLE progress when using Tock, but it was always pretty hacky unfortunately.

**Comment by idea--list** [comment]
I also would like to get some use of the on-chip BLE. I know it works with the Arduino BLE library + Arduino core v2.0.5 (which is based on Mbed core).

I wonder if we could discuss here exactly which part of the Mbed implementation is broken and also why. I think that way we could find a working solution a bit earlier. So which details are known until now?

**Comment by idea--list** [comment]
Seems Mbed OS V6.7 introduced a change that prohibits any code running on the Artemis Thing Plus (and maybe all the Artemis boards are affected). Mbed OS 6.8 RC1 solves this issue and even BLE seems to work, so i was wrong.

Am going through&testing all the BLE examples right now

**Comment by idea--list** [comment]
Mbed OS V6.8 has been released today, this release fixes the bug in V6.7 that prohibited any code running on Apollo3 chips.

Meanwhile i finished testing all the [official examples from Mbed](https://github.com/ARMmbed/mbed-os-example-ble). All the examples work except for the `BLE_PeriodicAdvertising` example. It states Periodic advertising is not supported by the chip.
Is that right or is this just a bug?

**Comment by dwebb72** [comment]
Does this mean the arduino examples will work now?
________________________________

**Comment by idea--list** [comment]
Yes, BLE examples of Arduino BLE library (V1.1.3) + Artemis Core (V2.0.5) work now with Arduino IDE (V1.8.13)

**Comment by idea--list** [comment]
Just checked the examples that come with the Arduino BLE library (V1.1.3) they also work using the Artemis core for Arduino (V2.0.5) and Arduino IDE (V1.8.13) but this has nothing to do with the official Mbed releases.

But actually also Mbed's BLE [examples](https://github.com/ARMmbed/mbed-os-example-ble) work except for periodic advertisments.

**Comment by idea--list** [comment]
As BLE seems to work as expected both with Arduino core V2.0.5 and Mbed OS V6.8 (most current releases of time of being), we could close this issue i think.

**Comment by truedat101** [comment]
> Mbed OS V6.8 has been released today, this release fixes the bug in V6.7 that prohibited any code running on Apollo3 chips.
> 
> Meanwhile i finished testing all the [official examples from Mbed](https://github.com/ARMmbed/mbed-os-example-ble). All the examples work except for the `BLE_PeriodicAdvertising` example. It states Periodic advertising is not supported by the chip. Is that right or is this just a bug?

Any idea about the BLE_PeriodicAdvertising?   What support is needed ?
